### DISCUSSION TITLE
How to update via choco

### DISCUSSION THREAD

**Discussion by mzfr** [ActivityWatch/activitywatch#628] [Q&A] [answered]
I've installed using `choco` package([link](https://community.chocolatey.org/packages/activitywatch/0.10.0)) but it hasn't been updated yet. So do I need to remove my current installation and then install again using windows downloader?

Also, I notice that the package isn't maintained by you guys, so do you plan to manage an official chocolatey package?

**Comment by ErikBjare** [comment, marked as answer]
An update has been submitted to Chocolatey (https://community.chocolatey.org/packages/activitywatch/0.11.0), but it's been put in the moderation queue due to some antivirus checks reporting false positives (#181).

We will likely not make an official Chocolatey package since we (the maintainers) don't use Windows ourselves. It's better than someone who actually uses Chocolatey maintains it.

**Reply by mzfr** [reply]
oh okay, thanks for the update.
